content/en/monitors/create/types/ci.md

```diff
@@ -19,18 +19,25 @@ further_reading:
 <div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-<div class="alert alert-info">CI Pipeline monitors are in alpha.
+<div class="alert alert-info">CI monitors are in alpha.
 </div>
 
 ## Overview
 
-Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline monitor to alert you when a specified type of CI Pipeline event exceeds a user-defined threshold over a given period of time.
+Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline monitor or a CI Test monitor.
+
+CI monitors allow you to visualize CI data and set up alerts on it. For example, use a CI Pipeline monitor to receive alerts on a failed pipeline or a job, and use a CI Test monitor to receive alerts on failed tests or slow tests.
 
 ## Monitor creation
 
-To create a [CI Pipeline monitor][2] in Datadog, use the main navigation: **Monitors -> New Monitor --> CI Pipelines**.
+To create a [CI Pipeline monitor][2] or [CI Test monitor][3] in Datadog, use the main navigation: *Monitors -> New Monitor --> CI*.
```

Comment:
```suggestion
To create a [CI monitor][2] in Datadog, use the main navigation: *Monitors -> New Monitor --> CI*.

Choose between a **Pipelines** or **Tests** monitor:
```

Similar to what APM has?

Reply:
indeed, it would be similar to APM monitor docs, but what link should we have to redirect the user to? `https://app.datadoghq.com/monitors/create/ci-pipelines`  or `https://app.datadoghq.com/monitors/create/ci-tests`?

Reply:
I guess it should be `https://app.datadoghq.com/monitors/create/ci-pipelines` since it's a first tab.